docs/platforms/react-native/session-replay/index.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Set Up Session Replay - Beta
+sidebar_order: 5500
+notSupported:
+description: "Learn how to enable the Beta of Mobile Session Replay in your app."
+---
+
+<Note>
+
+Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+
+If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-react-native/issues/new?assignees=&labels=Platform%3A+React-Native%2CType%3A+%F0%9F%AA%B2+Bug&projects=&template=BUG_REPORT.md) with a link to a relevant replay in Sentry if possible.
+
+</Note>
+
+[Session Replay](/product/explore/session-replay/) helps you get to the root cause of an error or latency issue faster by providing you with a reproduction of what was happening in the user's device before, during, and after the issue. You can rewind and replay your application's state and see key user interactions, like taps, swipes, network requests, and console entries, in a single UI.
+
+By default, our Session Replay SDK masks all text content, images, and user input, giving you heightened confidence that no sensitive data will leave the device. To learn more, see <PlatformLink to="/product/session-replay/">Session Replay Product docs</PlatformLink>.
+
+## Pre-requisites
+
+Make sure your Sentry React Native SDK version is at least 5.26.0.
+
+## Install
+
+If you already have the SDK installed, you can update it to the latest version with:
+```
+npm install @sentry/react-native --save
+```
```

Comment:
NIT: we could also mention the other was of installing it.
**NOTE**: I suggestion is breaking the format so you will need to replace `o`` by ```

```suggestion
`o``
npm install @sentry/react-native --save
`o``
`o``bash {tabTitle:npm}
npm install @sentry/react-native --save
`o``

`o``bash {tabTitle:yarn}
yarn add @sentry/react-native
`o``

`o``bash {tabTitle:pnpm}
pnpm add @sentry/react-native
`o``
```

Reply:
You can wrap it by adding 4 x `

Reply:
![image](https://github.com/user-attachments/assets/4b4ac3ef-f88d-45da-aa00-e8ea49fc3354)
like this